### ISSUE TITLE
macOS download not working

### ISSUE THREAD

**Issue by porres** [pd-l2ork/pd-l2ork#65]
Getting this

<img width="272" alt="Screenshot 2024-11-24 at 17 44 19" src="https://github.com/user-attachments/assets/8b08603d-e1fc-4273-8c5a-59482e383ce3">

**Comment by porres** [comment]
no, I attempted multiple times and it didn't work, I always get what I posted above...

And when downloading Pd-Vanilla from Miller's site, it works pretty nicely, this is what I get

<img width="284" alt="Screenshot 2024-11-25 at 13 14 20" src="https://github.com/user-attachments/assets/f9b2ea76-f5fd-4584-92e3-dcd64a6ae67d">

then I just hit 'ok' and it opens

**Comment by pd-l2ork** [comment]
What version are you using?

**Comment by porres** [comment]
I downloaded the latest from 
https://l2ork.music.vt.edu/main/make-your-own-l2ork/software/

I'm on macOS Sequoia 15.0.1 (24A348)

**Comment by porres** [comment]
oh, and I'm on an M3 macbook air

**Comment by pd-l2ork** [comment]
Thanks. Looking into this. In the meantime, after you get this prompt, you can go into System Preferences > Security > Open Anyway. Hope this helps.

**Comment by porres** [comment]
it might be the case, but I'm not sure of how it all works.

Purr Data seems to be going through the same thing and they explain how to bypass the system with some notes on what to do. Nothing too crazy, I guess a couple of commands on the terminal as sudo... check that out.

it really became ridiculous that they're asking you to pay so you can provide a download of a free/open software, that makes me sick :/

**Comment by pd-l2ork** [comment]
IIRC those commands disable checking any 3rd party software which could be a huge security hole. I would rather not do that and leave it up to the user to manually approve each app, as needed.
